wagn/lib/decko/engine.rb

```diff
@@ -20,7 +21,7 @@ class Engine < ::Rails::Engine
 
     paths.add "app/controllers", :with => 'rails/controllers', :eager_load => true
     paths.add 'gem-assets',      :with => 'rails/assets'
-    paths.add 'config/routes',   :with => 'rails/engine-routes.rb'
+    paths.add 'config/routes.rb',   :with => 'rails/engine-routes.rb'
```

Comment:
the 'config/routes' key is not used anymore in Rails 4 but 'config/routes.rb' seems to work.